Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -18,6 +18,26 @@ import PackageLoading
 
 @testable import Build
 
+#if os(macOS)
+    let defaultTargetTriple: String =  "x86_64-apple-macosx10.10"
+#elseif os(Windows)
+    let defaultTargetTriple: String =  "x86_64-unknown-windows-msvc"
+#elseif os(Linux)
+    #if arch(x86_64)
+      let defaultTargetTriple: String = "x86_64-unknown-linux"
```

Comment:
Please make this `x86_64-unknown-linux-gnu` ... exherbo is picky about the target triple since we use it for the separation of multiple architectures on a single root.

Reply:
While I agree with you that this should change to include `gnu`, SwiftPM currently uses `x86_64-unknown-linux` throughout, and we should make that adjustment in a different pull request.

Reply:
Tested after changing it to `x86_64-unknown-linux-gnu `, but got following error:

      Test Case 'BuildPlanTests.testBasicClangPackage' started at 2019-02-11 14:50:34.420
      /home/work/sw/master/swiftpm/Tests/BuildTests/BuildPlanTests.swift:304: error:   BuildPlanTests.testBasicClangPackage : XCTAssertEqual failed: ("["-target", "x86_64-unknown-linux", "-g", "-O0",  "-DSWIFT_PACKAGE=1", "-DDEBUG=1", "-fblocks", "-fmodules", "-fmodule-name=extlib", "-I", "/ExtPkg/Sources/extlib/include", "-fmodules-cache-path=/path/to/build/debug/ModuleCache"]") is not equal to ("["-target", "x86_64-unknown-linux-gnu", "-g", "-O0", "-DSWIFT_PACKAGE=1", "-DDEBUG=1", "-fblocks", "-fmodules", "-fmodule-name=extlib", "-I", "/ExtPkg/Sources/extlib/include", "-fmodules-cache-path=/path/to/build/debug/ModuleCache"]") -

Will keep it `x86_64-unknown-linux` ( "we should make that adjustment in a different pull request")